quantization_troubleshooting.md

```diff
@@ -0,0 +1,153 @@
+# Quantization Troubleshooting for MCT
+
+**Compressing a model with MCT:**
+The Model Compression Toolkit (MCT) has many features and capabilities for compressing your neural network with a minimal impact on accuracy.
+However, sometimes the compressed model's accuracy decrease is too large.
+But don't despair, the lost accuracy may be recovered by adjusting the quantization configuration
+or quantization setup.
+
+**Recover Accuracy: Troubleshooting:**
+The following list contains a set of operations that may recover lost accuracy do to compression with MCT.
+Some operations might be relevant to your model and others may not.
+
```

Comment:
General suggestion:
we are referring the users only to our keras tutorials. maybe add a sentence stating (somethinglike):

throughout this document we refer the user for notebooks that demonstrate how to use all the mentioned tools. We refer them to keras notebooks but all tools are also available in pytorch and some of them also have pytorch tutorials. you are welcome to check the complete set of notebooks to get a glimpse of the advanced features of MCT (and put a link)